apps/console/app/routes/_shell/sidebar-nav.tsx

```
@@ -0,0 +1,78 @@
+];
+
+type SidebarNavProps = {
+  activeAgent?: { slug: string };
+};
+
+export const SidebarNav = ({ activeAgent }: SidebarNavProps) => {
+  const { pathname } = useLocation();
+  const navigate = useNavigate();
+
+  navItems.forEach(({ shortcut, getPath }) => {
+    useHotkeys(
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)